src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -1,10 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>Microsoft.Spark</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
+
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <Description>.NET for Apache Spark</Description>
+    <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+    <PackageProjectUrl>https://github.com/dotnet/spark</PackageProjectUrl>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
This is already set in [https://github.com/dotnet/spark/blob/master/src/csharp/Directory.Build.props](https://github.com/dotnet/spark/blob/master/src/csharp/Directory.Build.props). No need to duplicate it here.

Reply:
removed. I also moved some other settings which I believe to all packages there.